src/main/java/se/bjurr/gitchangelog/internal/settings/SettingsJiraIssueFieldFilter.java

```diff
@@ -0,0 +1,39 @@
+package se.bjurr.gitchangelog.internal.settings;
+
+import static se.bjurr.gitchangelog.internal.util.Preconditions.checkNotNull;
+
+import java.io.Serializable;
+
+public class SettingsJiraIssueFieldFilter implements Serializable {
+  private static final long serialVersionUID = -658106272421601880L;
+  /** TODO: This could in theory be a defined list/enum of all available operators. */
```

Comment:
Yes, add enum.

Reply:
Sounds good, i'll do that in the follow-up PR that is essentially a fixed #97 and compatible with this stuff!